manifests/example-dev/replicator-modules.yaml

```diff
@@ -0,0 +1,6 @@
+parameters:
+  - name: eks-version
+    valueFrom:
+      envVariable: GLOBAL_EKS_VERSION
```

Comment:
Replace the `GLOBAL_EKS_VERSION` with values hardcoded and may be a commented line showing `how to use .env reference`